### ISSUE TITLE
declare parallel read/write safety for sphinx

### ISSUE THREAD

**Issue by JohannesSchilling** [svenevs/exhale#87]
in addition to the version, we should return `parallel_read_safe` and `parallel_write_safe` declarations from [`setup()`](https://github.com/svenevs/exhale/blob/master/exhale/__init__.py#L46) so that we avoid this warning:
```
the exhale extension does not declare if it is safe for parallel reading, assuming it isn't - please ask the extension author to check and make it explicit
```

now i don't know enough about the internals of exhale to know if parallel reading/writing would be safe, but as a start we could just set both to false, addressing the warning and leaving room for future improvement.

what do you think? happy to open a PR as well.

**Comment by svenevs** [comment]
Hi @JohannesSchilling thank you for reporting.  I put those in the wrong `setup` function :facepalm: I am going to be making some quick updates here and will push out a new release soon.

I'm very sorry for the delay, things are of course rather crazy right now :confused:

**Comment by JohannesSchilling** [comment]
thanks :-)

also (on a more personal note): absolutely no need to apologize for delays, thanks for working on this and providing this fine piece of software in the first place :-)

**Comment by svenevs** [comment]
closed in #126
